pkg/runtime/config.go

```diff
@@ -122,7 +123,6 @@ type internalConfig struct {
 	allowedOrigins               string
 	standalone                   configmodes.StandaloneConfig
 	kubernetes                   configmodes.KubernetesConfig
-	mTLSEnabled                  bool
```

Comment:
Please don't just move properties around as it risks causing merge conflicts unnecessarily.